### PR TITLE
Remove audience, platform, product, version, print

### PR DESCRIPTION
**Pull request by mjibson**
All unused.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,49 +1,11 @@
...
 homepage_title: CockroachDB Docs
 site_title: Cockroach Labs
 google_analytics: UA-63775601-1
 
-# variables
-sidebar_tagline: 
-sidebar_version: 
-project_file_name: doc
-
-port: 4001
```
Comment (jseldess) [inline _config.yml]: Could you put `port` back in? Although 4001 may be the default, it's nice to see how to change this, in cases where you want to serve up two sites at the same time.

Comment (mjibson) [inline _config.yml]: You set this when you invoke jekyll itself. It doesn't need to be in the config file. You can also do it via docker.

File: _data/sidebar_doc.yml [hunk 2]
```diff
@@ -4,249 +4,92 @@
...
-        - title: Table of Contents
-          url: /tocpage.html
-          audience: all
-          platform: all
-          product: all
-          version: all
-          web: false
-
     - title: Get Started
-      audience: all
```
Comment (jseldess) [inline _data/sidebar_doc.yml]: These tie into the theme's ability to tag content for specific outputs. So you can have one `md` file, for example, with conditional logic that makes it suitable for multiple outputs (paragraph 1 for version 1, paragraph 2 for version 2, etc.). This type of thing reduces duplication (i.e., unique copy of same file per version), which makes updating docs much more efficient. Long-term (and I know this isn't even remotely relevant for Cockroach), it dramatically reduces translation costs as well. 

Anyway, we're not using this now, so it's fine to remove. Once you get to thinking about how to handle versions, though, I'd like to talk it through and make sure our approach is as future-oriented as possible.

**Comment by jseldess** [comment]
@mjibson, I just got a local copy and served up the site. It's not rendering correctly. Not sure which changes caused this. 

**Docs home:**
screen shot 2016-02-25 at 1 18 45 pm

**Install page:**
screen shot 2016-02-25 at 1 19 47 pm

File: _config.yml (hunk 1 above)
Comment (jseldess) [inline _config.yml]: OK, can you point me to an example so I know how to do this going forward?

Comment (mjibson) [inline _config.yml]: https://jekyllrb.com/docs/configuration/

The `--port 1234` option when you start jekyll does it.

**Comment by mjibson** [comment]
I added the defaults back in. Works again.

**Comment by jseldess** [comment]
Great. LGTM.

**Pull request merged**